static/app/components/group/groupSummary.tsx

```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
+            </Fragment>
+          )}
+          {data && expanded && <Headline>{data.headline}</Headline>}
+        </CollapsedRow>
+        <IconContainerRight>
+          <IconChevron direction={expanded ? 'up' : 'down'} />
+        </IconContainerRight>
+      </StyledTitleRow>
+      {expanded && (
+        <Body>
+          <div>
+            {isError ? <div>{t('Error loading summary')}</div> : null}
+            {data && (
+              <Content>
                 <SummaryContent
                   dangerouslySetInnerHTML={{
-                    __html: marked(data.impact),
+                    __html: marked(`**Details:** ${data.summary}`),
                   }}
                 />
-              </ImpactContent>
-            </Content>
+                <ImpactContent>
+                  <SummaryContent
+                    dangerouslySetInnerHTML={{
+                      __html: marked(`**Impact:** ${data.impact}`),
+                    }}
+                  />
+                </ImpactContent>
+              </Content>
+            )}
+          </div>
+          {openForm && !isPending && (
+            <ButtonContainer>
+              <Button
+                onClick={() => {
+                  openForm({
+                    messagePlaceholder: t(
+                      'How can we make this issue summary more useful?'
+                    ),
+                    tags: {
+                      ['feedback.source']: 'issue_details_ai_issue_summary',
+                      ['feedback.owner']: 'ml-ai',
+                    },
+                  });
+                }}
+                size="xs"
+                icon={<IconMegaphone />}
+              >
+                Give Feedback
+              </Button>
+              <GroupSummaryFeatureBadge />
+            </ButtonContainer>
           )}
-        </div>
-        {openForm && !isPending && (
-          <ButtonContainer>
-            <Button
-              onClick={() => {
-                openForm({
-                  messagePlaceholder: t(
-                    'How can we make this issue summary more useful?'
-                  ),
-                  tags: {
-                    ['feedback.source']: 'issue_details_ai_issue_summary',
-                    ['feedback.owner']: 'ml-ai',
-                  },
-                });
-              }}
-              size="xs"
-              icon={<IconMegaphone />}
-            >
-              Give Feedback
-            </Button>
-          </ButtonContainer>
-        )}
-      </Wrapper>
-    </SidebarSection.Wrap>
+        </Body>
+      )}
+    </Wrapper>
   );
 }
 
+const Body = styled('div')`
+  padding-left: ${space(3)};
+  padding-right: ${space(3)};
+  padding-top: ${space(1)};
+  padding-bottom: ${space(1.5)};
+`;
+
+const Headline = styled('p')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: ${space(0.5)};
+  flex-shrink: 0;
+  margin-top: ${space(3)};
```

Comment:
margin here is really odd. why is it needed?

![image](https://github.com/user-attachments/assets/1aa61739-4c8f-49df-b71d-f57c5f49feb1)